lib/logstash/codecs/cef.rb

```diff
@@ -155,36 +274,21 @@ def handle(data, &block)
     # Get rid of the CEF bit in the version
     event.set('cefVersion', event.get('cefVersion').sub(/^CEF:/, ''))
 
-    # Strip any whitespace from the message
-    if not message.nil? and message.include? '='
+    # Use a scanning parser to capture the Extension Key/Value Pairs
+    if message && !message.empty? && message.include?('=')
```

Comment:
wondering if doing `!message.empty? && message.include?('=')` vs just  `message.include?('=')` in actually faster ?  - I mean if most `message` are expected to be non empty at this point I'd assume this extra `!message.empty?` call will be actually slower and wonder if for actual empty `message` checking for emptiness before calling  `include?` has a practical benefit?